modules/photon/photon.js

```diff
@@ -1,54 +1,60 @@
 /* jshint onevar: false */
 
-( function( $ ) {
+( function() {
+	function recalculate() {
+		if ( this.complete ) {
+			// Support for lazy loading: if there is a lazy-src attribute and it's value
+			// is not the same as the current src we should wait until the image load event
+			var lazySrc = this.getAttribute('data-lazy-src');
+			if ( lazySrc && this.src !== lazySrc ) {
+				this.addEventListener( 'load', recalculate );
+				return;
+			}
+
+			// Copying CSS width/height into element attributes.
+			var width = this.width;
+			var height = this.height;
+			if ( width && width > 0 && height && height > 0 ) {
+				this.setAttribute('width', width);
+				this.setAttribute('height', height);
+
+				reset_for_retina( this );
+			}
+		}
+		else {
+			this.addEventListener( 'load', recalculate );
+			return;
+		}
+	}
 	/**
 	 * For images lacking explicit dimensions and needing them, try to add them.
 	 */
 	var restore_dims = function() {
-			$( 'img[data-recalc-dims]' ).each( function recalc() {
-				var $this = $( this );
-				if ( this.complete ) {
-					// Support for lazy loading: if there is a lazy-src
-					// attribute and it's value is not the same as the current src we
-					// should wait until the image load event
-					if ( $this.data( 'lazy-src' ) && $this.attr( 'src' ) !== $this.data( 'lazy-src' ) ) {
-						$this.load( recalc );
-						return;
-					}
-
-					var width = this.width,
-						height = this.height;
-
-					if ( width && width > 0 && height && height > 0 ) {
-						$this.attr( {
-							width: width,
-							height: height,
-						} );
+		var elements = document.querySelectorAll( 'img[data-recalc-dims]' );
```

Comment:
Note that `querySelectorAll` returns `NodeList`, not an array: https://developer.mozilla.org/en-US/docs/Web/API/NodeList

This is important in case anyone tries to loop it later with `forEach` or similar; it would break on IE11. Looping it with `for` [seems to be safe tho](https://developer.mozilla.org/en-US/docs/Web/API/NodeList#Example), so no changes required here. Just wanted you to be aware in case it wasn't conscious decision to use `for` here.

Reply:
And most importantly, do test with IE11 just in case. :-)